clang/lib/Format/WhitespaceManager.cpp

```diff
@@ -268,11 +380,24 @@ FindPrevPointerOrReference(SmallVector<WhitespaceManager::Change, 16> &Changes,
   return i;
 }
 
+enum {
```

Comment:
Is there a feasible way to enforce type-safety here, and/or avoid mixing control/state flags in a single enum?